images/Dockerfile

```diff
@@ -1,25 +1,30 @@
 # Source: https://github.com/dotnet/dotnet-docker
 FROM mcr.microsoft.com/dotnet/runtime-deps:6.0-jammy as build
 
+ARG TARGETOS
+ARG TARGETARCH
 ARG RUNNER_VERSION
-ARG RUNNER_ARCH="x64"
 ARG RUNNER_CONTAINER_HOOKS_VERSION=0.3.2
+ARG RUNNER_ARCH="x64"
```

Comment:
do we still this as an ARG?

Reply:
I think it should be safe to remove it
With this change, it is calculated always, and I tested it on my fork just in case